images/template-service-broker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# This image will run the template service broker.
+# The entry point name is not yet final.
+
+FROM openshift/origin
+
+ENTRYPOINT ["/usr/bin/openshift start template-service-broker"]
```

Comment:
> Doesn't this need to be an actual list?

quite possibly/probably.  Are there other metadata files I'd need to touch for a new image?